prime-angular-frontend/src/app/shared/components/forms/college-certification-form/college-certification-form.component.ts

```diff
@@ -382,4 +412,17 @@ export class CollegeCertificationFormComponent implements OnInit {
   private filterPractices(collegeCode: number): PracticeConfig[] {
     return this.practices.filter(p => p.collegePractices.map(cl => cl.collegeCode).includes(collegeCode));
   }
+
+
+  private checkLicenseIfDiscontinued() {
+    if (this.collegeCode.value && this.licenseCode.value) {
+      this.licenseClassDiscontinued = this.isCertificationDiscontinued(this.collegeCode.value, this.licenseCode.value);
+      this.collegeDiscontinued = this.colleges.some(c => c.code == this.collegeCode.value);
```

Comment:
There is something not clear about this line ... also should that be a triple equals?

Reply:
Comment added and changed to a triple equals. Also, modified the logic a bit for your comment below.